app/services/batched_remove_status_service.rb

```diff
@@ -47,10 +47,9 @@ def call(statuses, **options)
 
     # Cannot be batched
     @status_id_cutoff = Mastodon::Snowflake.id_at(2.weeks.ago)
-    redis.pipelined do
-      statuses.each do |status|
-        unpush_from_public_timelines(status)
-      end
+
+    statuses.each do |status|
+      unpush_from_public_timelines(status)
```

Comment:
We probably want to still pipeline it, just pass the pipeline as an argument to `unpush_from_public_timelines` or something.